core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -182,6 +184,12 @@ private[ui] class StagePage(parent: StagesTab, store: AppStatusStore) extends We
               {Utils.bytesToString(stageData.diskBytesSpilled)}
             </li>
           }}
+          {if (!stageJobIds.isEmpty) {
+            <li>
+              <strong>Associated Job Ids: </strong>
+              {stageJobIds}
```

Comment:
make it href link?

Reply:
The problem here is that the stage could also have multiple job ids, in that case, we get a bunch of them. Do you want a generic link instead that will take you to the jobs page? Let me know what you think.

Reply:
My suggestion is to map each job id as a href link.

Reply:
Did that and updated the screenshot. Thank you.